doc/source/parameters/dem/insertion_info.rst

```diff
@@ -133,4 +138,12 @@ Each line is associated with a particle and its properties. The ``fem_force`` an
     The ``file`` insertion combine with the ``extract-particles-properties-from-vtu.py`` python code can be a useful tool. The loading of particles and the rest of the simulation can be performed in two different triangulations, witch is not the case of the the restart feature. This means that the loading triangulation can have smaller cells and a bigger domain to allow for the use of larger insertion boxes. Then, particles properties can be extracted and the remainder of the simulation can be performed in the appropriate triangulation.
 
 .. warning::
-    The critical Rayleigh time step is computed from the parameters in the ``particle type`` subsections, not the ``insertion info`` subsection. It is the user's responsibility to fill the ``particle type`` subsections correctly according to the diameter values stored in the insertion input file, otherwise Rayleigh time percentage displayed at the start of every DEM simulation may not be accurate.
\ No newline at end of file
+    The critical Rayleigh time step is computed from the parameters in the ``particle type`` subsections, not the ``insertion info`` subsection. It is the user's responsibility to fill the ``particle type`` subsections correctly according to the diameter values stored in the insertion input file, otherwise Rayleigh time percentage displayed at the start of every DEM simulation may not be accurate.
+
+---------------------
+Clear and add
+---------------------
+Just like the ``file`` insertion method, the ``clear_and_add`` insertion method insert particles using the same type of external files. Two main differences can be noted between these methods. First, with the ``clear_and_add`` method, a list of files can be define using the ``list of input files`` parameter. Each insertion time step, a different file will be used to insert the particles from this list of files in sequential order. If the end of the list is reached and there is still particles left to be inserted, the next file being used will be the first one in that list. The second difference between these two methods regard the clearing feature of the ``clear_and_add`` insertion method. At every insertion time step, all the particles located inside a predefined box will be deleted. This box can be defined using the ``clearing box points coordinates`` parameter which works exactly like the ``insertion box`` for the ``volume`` insertion method. The clearing is done before the insertion from a given file, thus there is no risk of inserting and deleting the same particle in the same insertion time step. If used properly, this method can be used to clear a certain zone of the simulation domain and add particle in that same zone right away. This assure that there is no excessive overlap between particle on insertion.
+
+.. note::
+    The ``clear_and_add`` insertion method shouldn't be used for most application. It can be extremely useful for really specific DEM simulation, but remain an experimental feature.
```

Comment:
```suggestion
    The ``clear_and_add`` insertion method is not suited to most applications. It can be useful for some DEM simulations, but this remains an experimental feature.
```

Reply:
What applications, for example? I would add examples here, since you don't have an example of use in the documentation.

Reply:
Additionally, it is important to mention that the feature, as is, is limited to a hexahedron.